client/lib/posts/post-list-cache-store.js

```diff
@@ -34,6 +35,10 @@ function getCacheKey( options ) {
 }
 
 function get( query ) {
+	if ( config.isEnabled( 'sync-handler' ) ) {
+		return console.warn( 'synchandler is enabled. Disable post-list-cache-store' );
```

Comment:
Yes, I agree. I've disabled just to make some proofs. Let's enable again and start to work in the double callback here as well.